reframe/core/pipeline.py

```diff
@@ -1691,7 +1691,10 @@ def _create_job(self, job_type, force_local=False, **job_opts):
                           name=f'rfm_{self.short_name}',
                           script_filename=script_name,
                           workdir=self._stagedir,
-                          sched_access=self._current_partition.access,
+                          sched_access=(
+                              list(self._current_partition.access) +
+                              list(self._current_environ.sched_bind_options)
+                          ),
```

Comment:
I would rather not mix it with `sched_access`. Instead, I would prepend it to the `sched_options` passed in `job_opts`.